contracts/deploy/051_oeth.js

```diff
@@ -0,0 +1,273 @@
+const { deploymentWithGuardianGovernor } = require("../utils/deploy");
+const addresses = require("../utils/addresses");
+const hre = require("hardhat");
+const { BigNumber, utils } = require("ethers");
+const {
+  getAssetAddresses,
+  getOracleAddresses,
+  isMainnet,
+  isFork,
+  isMainnetOrFork,
+} = require("../test/helpers.js");
+
+// 5/8 multisig
+const guardianAddr = addresses.mainnet.Guardian;
+
+module.exports = deploymentWithGuardianGovernor(
+  { deployName: "051_oeth", forceDeploy: true },
+  async ({ deployWithConfirmation, ethers, getTxOpts, withConfirmation }) => {
+    const { deployerAddr, governorAddr } = await getNamedAccounts();
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+
+    // actions = actions.concat(actions2)
+    let actions = await deployCore({
+      deployWithConfirmation,
+      withConfirmation,
+      ethers,
+    });
+
+    await deployDripper({ deployWithConfirmation, withConfirmation, ethers });
+
+    actions = actions.concat(
+      await deployFraxETHStrategy({
+        deployWithConfirmation,
+        withConfirmation,
+        ethers,
+      })
+    );
+
+    // Governance Actions
+    // ----------------
+    return {
+      name: "Deploy OETH Vault, Token, Strategies, Harvester and the Dripper",
+      actions,
+    };
+  }
+);
+
+/**
+ * Deploy the core contracts (Vault and OETH).
+ */
+const deployCore = async ({
+  deployWithConfirmation,
+  withConfirmation,
+  ethers,
+}) => {
+  const { deployerAddr } = await getNamedAccounts();
+  const sDeployer = await ethers.provider.getSigner(deployerAddr);
+
+  const assetAddresses = await getAssetAddresses(hre.deployments);
+  console.log(
+    `Using asset addresses: ${JSON.stringify(assetAddresses, null, 2)}`
+  );
+
+  // Proxies
+  await deployWithConfirmation("OETHVaultProxy");
+  await deployWithConfirmation("OETHProxy");
```

Comment:
I thought we had already deployed the OETH proxy?

Reply:
You're right nice catch thanks. Removing this line